ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -52,23 +81,48 @@ def __init__(
 
         self_play_parameters = trainer_parameters["self_play"]
         self.window = self_play_parameters.get("window", 10)
-        self.play_against_current_self_ratio = self_play_parameters.get(
```

Comment:
You should make sure this is between 0 and 1

Reply:
Raising a warning